src/components/MultiCombobox/MultiCombobox.tsx

```diff
@@ -11,6 +11,13 @@ import { typedMemo } from '../../utils/helpers';
 import Menu from '../utils/Menu';
 import AbstractButton from '../AbstractButton';
 import ComboBoxItems from '../utils/ComboBoxItems/ComboBoxItems';
+import Icon from '../Icon';
+
+type DefaultContentProps<T> = {
```

Comment:
nit: can we define this as `Pick<MultiComboboxProps<T>, 'value' | 'removeItem' | 'itemToString'>`? Just so we don't redefine typings...